src/ZODB/blob.py

```diff
@@ -942,13 +942,14 @@ def remove_committed_dir(path):
                 filename = os.path.join(dirpath, filename)
                 remove_committed(filename)
         shutil.rmtree(path)
-
-    link_or_copy = shutil.copy
+        link_or_copy = shutil.copy
```

Comment:
This assignment **must** be at the same indentation level as the function definitions:  it is not part of the function.

```suggestion
    link_or_copy = shutil.copy
```